src/tools/genpolicy/Cargo.toml

```diff
@@ -5,7 +5,7 @@
 
 [package]
 name = "genpolicy"
-version = "0.1.0"
+version = "3.2.0-1"
```

Comment:
Any chance we can do something closer to the way version gets handled upstream? E.g.,

/usr/local/bin/containerd-shim-kata-cc-v2 --version
Kata Containers containerd shim (Golang): id: "io.containerd.kata.v2", version: 3.3.0-alpha0, commit: 9b7dcc0d98cf138e519d65cac8ef52ae9f42a5fb

Reply:
Yea, I'll take a look

Reply:
Added